daemon/pod.go

```diff
@@ -195,6 +196,8 @@ func (p *Pod) InitContainers(daemon *Daemon) error {
 
 		config := &container.Config{
 			Image:           c.Image,
+			Entrypoint:      strslice.New(c.Entrypoint...),
```

Comment:
I saw  the pr https://github.com/hyperhq/hyper/pull/201 check the entrypoint before assign it to config.Entrypoint

+               if len(c.Entrypoint) != 0 {
+                       config.Entrypoint = strslice.New(c.Entrypoint...)
                }